changelog.md

```diff
@@ -1,4 +1,25 @@
 
+## 3.1.0 / 2020-09-14
+
+### New Features
+
+- A notification wall is now available in header bar. You can now seen all new activities done in Tracim.
+- It is now possible to mention user directly with his Tracim username like this `@username`. All mention are visible in notification wall.
```

Comment:
```suggestion
- It is now possible to mention user directly with his Tracim username like this `@username`. All mentions are visible in notification wall.
```

Reply:
Wording change. thanks for your proposal.